slides/_naming_for_clarity.qmd

```diff
@@ -18,4 +18,53 @@ A few naming tips and conventions:
 
 ## Naming For Clarity
```

Comment:
I would include a `{.smaller}` here for consistency with the rest of the slides:
```suggestion
## Naming For Clarity {.smaller}
```